socorro/processor/mozilla_transform_rules.py

```diff
@@ -391,89 +391,21 @@ def _action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
         return True
 
 
-def setup_product_id_map(config, local_config, args_unused):
-    database_connection = local_config.database_class(local_config)
-    transaction = local_config.transaction_executor_class(
-        local_config,
-        database_connection
-    )
-    sql = (
-        "SELECT product_name, productid, rewrite FROM "
-        "product_productid_map WHERE rewrite IS TRUE"
-    )
-    product_mappings = transaction(
-        execute_query_fetchall,
-        sql
-    )
-    product_id_map = {}
-    for product_name, productid, rewrite in product_mappings:
-        product_id_map[productid] = {
-            'product_name': product_name,
-            'rewrite': rewrite
-        }
-    return product_id_map
-
-
 class ProductRewrite(Rule):
```

Comment:
If I had my druthers, the processor rules would have documentation about wtf they're doing. For example, this rule stomps on the ProductName but I have no idea why.

We don't have to do that in this PR, but if it's the case that this sort of thing is in your head, might as well throw that into a docstring now.